src/pages/hegel/reference/mechanical-object/section-1.mdx

```diff
@@ -4,18 +4,21 @@ description: Learn about Section 1 of the Mechanical Object
 ---
 
 
-The first section opens with a paragraph that is a reflection on the “mechanical object” rather than a continuation of the immanent development of what the “mechanical object” is, as was the case in the previous paragraph. This is something that Hegel does often, and without any warning, and it is important to distinguish passages that add to the immanent development of a concept from passages that are reflections on that immanent development.  
+The first section opens with a paragraph that is a reflection on the ```mechanical object``` rather than a continuation of the immanent development of what the ```mechanical object``` is, as was the case in the previous paragraph. This is something that Hegel does often, and without any warning, and it is important to distinguish passages that add to the immanent development of a concept from passages that are reflections on that immanent development.  
```

Comment:
For inlined code elements, just enclose the words or words between a single pair of backticks, like so `mechanical object`.

Triple backticks are used for code paragraphs.

```
This is a code paragraph.
```

Reply:
Thank you. Made the changes.